source/pages/_document.tsx

```diff
@@ -1,14 +1,42 @@
 import Document, { Head, Html, Main, NextScript } from 'next/document';
+import React from 'react';
+import { CardanoNetwork } from '../constants';
+import { environment } from '../environment';
 import { DEFAULT_LOCALE } from '../features/i18n';
 import { cardanoExplorerTheme } from '../styles/theme/theme';
 
 class CardanoExplorerDocument extends Document {
   public render() {
     const lang =
       this.props.__NEXT_DATA__.props?.pageProps?.locale || DEFAULT_LOCALE;
+
+    const injectGA = () => {
+      if (typeof window === 'undefined') {
+        return;
+      }
+      const windowObject: Window | any = window;
+      windowObject.dataLayer = windowObject.dataLayer || [];
+      function gtag(param: string, value: any, config?: any) {
```

Comment:
- There should be a better type to assign `value`
- Is `config` and unused param?

Reply:
Fixed